app/js/services/situationService.js

```diff
@@ -115,8 +115,12 @@ angular.module('ddsCommon').factory('SituationService', function($http, $session
             situation.individus.forEach(function(individu) {
                 individu.date_naissance = individu.date_naissance.format('YYYY-MM-DD');
                 delete individu.hasRessources;
+
+                if (individu.dateDernierContratTravail) {
```

Comment:
Maybe `jsyaml` has a way to always serialize instances of `Moment.js` into a certain format?